src/main/java/lesson2/concerttickets/utils/NullableWarningManager.java

```diff
@@ -0,0 +1,30 @@
+package lesson2.concerttickets.utils;
+
+import lesson2.concerttickets.model.annotation.NullableWarning;
+
+import java.lang.reflect.Field;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class NullableWarningManager {
```

Comment:
This could be done without separated class, checkFieldsForNull() could be located  within parent Entity class, and then you just should call it in children's constructor eg. in Ticket class